make/DynamoRIOConfig.cmake.in

```diff
@@ -988,15 +989,15 @@ function (configure_DynamoRIO_client target)
     if (APPLE)
       set(LD_FLAGS "-arch x86_64 -image_base ${PREFERRED_BASE}")
     elseif (UNIX)
-      if (LINKER_IS_GNU_GOLD)
+      if (LINKER_IS_GNU_GOLD OR LINKER_IS_LLVM_LLD)
         # Gold doesn't have a default version script for us to edit.  However,
         # it has a handy command line flag that does exactly what we want.  Note
         # that gnu ld has -Ttext as well, but it is very different.
         # XXX: gnu ld added an equivalent -Ttext-segment to binutils 2.20 in
         # 2009.  We could switch to that if we ever drop support for old
         # linkers.
         set(PREFERRED_BASE_FLAGS "-Wl,-Ttext=${PREFERRED_BASE}")
-      else (LINKER_IS_GNU_GOLD)
+      else (LINKER_IS_GNU_GOLD OR LINKER_IS_LLVM_LLD)
```

Comment:
The condition in the endif() below doesn't match this. It causes some build warnings. #7126 fixes it.